sample-docker-app/package.json

```
@@ -11,7 +11,9 @@
     "build": "tsc",
     "start": "ts-node index",
     "test": "echo \"Error: no test specified\" && exit 1",
-    "build:image": "npm run build && docker build -t sample-docker-app ."
+    "build:images": "npm run build && npm run build:image:x86_64 && npm run build:image:arm64v8",
```

@Alexander89 what do you think? We’d need to update the instructions on the website, right?

---

For me, it would be better only to add a second command for ARM64. 

But that means, having two ax-manifests.yml. One for people who like to build it for arm and the other one who just like to run it on the local machine. The crossbuild requires some extrawork on windows and some linux distribution. (e.g.: I use Fedora)

```suggestion
    "build:image": "npm run build && npm run build:image:x86_64"
    "build:image:arm": "npm run build && npm run build:image:arm64v8",
```

And yes, we are in the nice position to add a ARM64 section at the tutorial.

---

I couldn't find how to specify the manifest path to `ax`, could you tell me ?

---

you find the complete documentation about the ax_cli here:
- https://developer.actyx.com/docs/cli/
- https://developer.actyx.com/docs/cli/apps/package